.nf-core.yml

```diff
@@ -21,6 +23,7 @@ lint:
     - .github/ISSUE_TEMPLATE/feature_request.yml
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/workflows/branch.yml
+    - .github/workflows/linting.yml
```

Comment:
Is this due to an issue with the `nf_core_version` bump?

Reply:
Good catch. I think this made it in while merging your branch with mine. I removed this line since it's not needed to ignore this file in linting. Fixed in 42fa0f87a600ebd19b003437ee0743f2852bec5b (along with some additions to the CHANGELOG)